.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/../../../../../../../:\Users\juanr\Desktop\WayfieldProject\seleniumscripter\.idea/dataSources/
```

Comment:
Dunno why a full crazy path exists here...

Reply:
Suggested edit: `**/dataSources` Automatically traverses all directories and excludes anything named `dataSources`